assets/src/edit-story/components/animationTimeline/stories/index.js

```diff
@@ -30,9 +30,9 @@ const animations = Array.from(Array(10).keys()).map((id) => ({
 }));
 
 export const _default = () => {
-  return <AnimationTimeline animations={animations} />;
+  return <AnimationTimeline animations={animations} duration={320} />;
 };
 
 export const noAnimations = () => {
-  return <AnimationTimeline animations={[]} />;
+  return <AnimationTimeline animations={[]} duration={500} />;
```

Comment:
How does `duration` correlate to the timeline.  Looking at storybook, the `noAnimations` timeline extends to about 8.5s (with a duration of 500), and the `Default` timeline extends to about 5.5s (with a duration of 320).

My assumption is that duration would be in `ms` but it doesn't seem that way 🤔

Reply:
You're right. I confused minutes and seconds. Updating.